example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -102,9 +102,6 @@ describe('The Ingest Granule failure workflow', () => {
       await deleteGranule({
         prefix: config.stackName,
         granuleId: inputPayload.granules[0].granuleId,
-        pRetryOptions: {
-          retries: 0,
-        },
```

Comment:
I'm not sure about this. Need to understand what's happening here.

The first response we get back from deleteGranule is a 403 Access Denied. If we do NOT retry, that throws and fails the test. If we DO retry, it starts returning 404, which does not cause the test to fail. Removing this block allows retries and fixes the test but I'm not sure what changed here.

Reply:
Resolved. We don't need to update the retry behavior here: https://github.com/nasa/cumulus/pull/3256/commits/4fff6333ef18056ed959517788f6c58aa0e08590

The test cleanup was fixed here: https://github.com/nasa/cumulus/pull/3256/commits/c1339da5bb078183bf435c85d4155dc8fd226b9c